src/disemvowel.js

```diff
@@ -0,0 +1,4 @@
+export const disemvowel = (string) => {
```

Comment:
Thank you guys for the feedback! I totally misinterpreted the instructions lol. I will adjust the function for implicit return and update the RegExp before merging this branch to master.